File: 2026-09/src/day_11/prms_181861/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> v;
    for (int x : arr) {
        for (int i = 0; i < x; i++) {
            v.push_back(x);
        }
    }

    return v;
}

File: 2026-09/src/day_11/prms_181861/Solution.java
package day_11.prms_181861;

import java.util.*;

class Solution {
    public int[] solution(int[] arr) {
        List<Integer> list = new ArrayList<>();
        for (int x : arr) {
            for (int i = 0; i < x; i++) {
                list.add(x);
            }
        }

        return list.stream().mapToInt(Integer::intValue).toArray();
    }
}
